### PR TITLE
Add command palette support

### PR DESCRIPTION
**Pull request by Kristinita**
In my opinion, people should not make the setting before starting to use the plugin. I believe, that the plugin should be used at once after installation.

Thanks.

### CODE REVIEW

File: Default.sublime-commands [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
```
Comment (facelessuser) [inline Default.sublime-commands]: I would probably prefer:

- Find Cursor
- Find Cursor Reverse
- Find Cursor Iterative
- Find Cursor Iterative Reverse

If I could go back, I would probably rename `pan` to `iterative`.

File: Default.sublime-commands [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+				"caption": "Find Cursor: Reverse False, Pan True",
+				"command": "find_cursor",
+				"args": {"reverse": false, "pan": true}
+			},
+			{
+				"caption": "Find Cursor: Reverse True, Pan True",
+				"command": "find_cursor",
+				"args": {"reverse": true, "pan": true}
+			},
+]
```
Comment (facelessuser) [inline Default.sublime-commands]: Please add new line at the end of the file.

File: Default.sublime-commands [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+			{
+				"caption": "Find Cursor: Reverse False, Pan True",
+				"command": "find_cursor",
+				"args": {"reverse": false, "pan": true}
+			},
+			{
+				"caption": "Find Cursor: Reverse True, Pan True",
+				"command": "find_cursor",
+				"args": {"reverse": true, "pan": true}
+			},
```
Comment (facelessuser) [inline Default.sublime-commands]: Please remove this trailing comma.  JSON format doesn't like trailing commas (unnecessary commas).

Comment (Kristinita) [inline Default.sublime-commands]: All right. But Sublime Text configuration files supported trailing commas. In my opinion, trailing comma it has necessity only if JSON checked for validity.

File: Default.sublime-commands (hunk 2 above)
Comment (Kristinita) [inline Default.sublime-commands]: [All right](http://stackoverflow.com/a/729795/5951529).

File: Default.sublime-commands (hunk 3 above)
Comment (facelessuser) [inline Default.sublime-commands]: True, Sublime doesn't care, but I run a format check on all Sublime settings and JSON files.  And currently, the trailing comma is breaking continuos validation.  It keeps things consistent, even if it is a bit silly.

**Comment by facelessuser** [comment]
As noted in #4, I mixed up iterative mode with panning mode (not sure why I though they were the same).  Also noted in #4, due to the slow nature of running commands through the palette, it probably doesn't make sense to include iterative in the palette.  Iterative mode is best used as a shortcut.  I've always used FindCursor as a shortcut.

**Comment by facelessuser** [comment]
Now that I am well rested, I went ahead and updated documentation to clarify things better.  I also added the command palette changes, but I only added panning mode as iterative doesn't really play nice from the command palette.  To use iterative mode, I recommend creating a shortcut command to quickly jump between all cursors.  Sorry for the confusion...this is what happens when I answer questions and review code late at night :sleepy:.

**Pull request closed without merging**